product_docs/docs/pem/8.1.1/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: "EDB Postgres Enterprise Manager"
+directoryDefaults:
+  description: "EDB Postgres Enterprise Manager Version 8 Documentation and release notes. PostgreSQL GUI tool for monitoring and performance optimization."
+---
+
+Welcome to Postgres Enterprise Manager (PEM). Postgres Enterprise Manager (PEM) consists of components that provide the management and analytical functionality for your EDB Postgres Advanced Server or PostgreSQL database. PEM is based on the Open Source pgAdmin 4 project.
+
+pgAdmin 4 is the leading Open Source management tool for Postgres, the world's most advanced Open Source database. pgAdmin 4 is a comprehensive [database](http://www.postgresql.org) design and management system. pgAdmin 4 is designed to meet the needs of both novice and experienced Postgres users alike, providing a powerful graphical interface that simplifies the creation, maintenance and use of database objects.
+
+!!! Note
+    This release is a re-release of 8.1.0 with a REST API bug fix. For the latest documentation, refer to [version 8.1.0](https://www.enterprisedb.com/docs/pem/8.1.0).
```

Comment:
@josh-heyer, best practice is for this to be a relative path, correct? I believe the path can be ../8.1.0, correct?

Reply:
@drothery-edb, I have changed the url with relative path. Also, tested on local build and have committed.